README.md

```diff
@@ -8,6 +8,7 @@
 ### Prerequisites
 
 - Node v6.0.0 or higher
+- [Kubernetes][kubectl] or [Docker][docker]
```

Comment:
This part I'm not super pleased with because it links to the kubectl page instead of the generic kubernetes website, but I feel like the kubectl page is useful, and just by looking at the kubernetes quickstart guide it isn't super easy to find. If anybody can think of a better way to link to that page please let me know!